roles/reproducer/tasks/network_assertions.yml

```diff
@@ -47,6 +47,33 @@
               defined networks. Those nets don't seem to be attached:
               {{ _defined | difference(_all_nets) }}
   rescue:
+    - name: Dump initial layout
+      ansible.builtin.debug:
+        var: cifmw_libvirt_manager_configuration
+
+    # This param exists only in case of CI reproducer run
+    # so it's expected to usually see it as "undefined" in
+    # the debugging output.
+    - name: Dump CI reproducer generated layout
```

Comment:
how about adding a `when: cifmw_libvirt_manager_configuration_gen is defined` ?

Reply:
it would just output "UNDEFINED" or something like that. No failure, so imho we can go as-is

Reply:
but why print a value if it's undefined? :)

Reply:
filtered both (default + gen) according to the inline comment.